### PR TITLE
Phoenix Update and Drive Zero

### PR DESCRIPTION
**Pull request by KianChilds**
## Justification
Fixes issue #2 

## Implementaion
Updates to latest version, and zeroed using phoenix X tool.

## Testing Done
Drive Train Drives

### CODE REVIEW

**Review by azvanderpas — APPROVED** [review]
Successful test on field -> Approved

**Comment by zbrotherton** [comment]
Let's add a TODO to change it back to canivore when possible

**Comment by zbrotherton** [comment]
Also can we pull the intake code off of this PR?
Just so we can get swerve into main

Have we decided - are we making alphabot a separate main? Should we make this branch protected from direct commits?

File: src/main/java/frc/robot/commands/IntakeCommand.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+public class IntakeCommand extends Command {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final IntakeSubsystem intakeSubsystem;
+
+  /**
+   * Creates a new ExampleCommand.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public IntakeCommand(IntakeSubsystem intakeSubsystem) {
```
Comment (lohengrin332) [inline src/main/java/frc/robot/commands/IntakeCommand.java:20]: Please make all method parameters and variables `final` wherever possible.

File: src/main/java/frc/robot/commands/IntakeCommand.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    intakeSubsystem.setIntakeSpeed(-0.75);
```
Comment (lohengrin332) [inline src/main/java/frc/robot/commands/IntakeCommand.java:33]: Should this value be a constant, either here in this class or in our main constants class?

**Review by lohengrin332 — APPROVED** [review]

**Pull request merged**